### DISCUSSION TITLE
channel listener on specific field only

### DISCUSSION THREAD

**Discussion by pierroo** [supabase/supabase#20597] [Questions]
I setup a channel on update on a specific table; and I was wondering if there is a way to ONLY listen to changes on a specific field of this table/row.

I have for example a chat_room, whose "last_msg" timestamp gets updated on each message inserted in a related chat_msg table.
I wanna listen only to a specific field "isTyping" for example (for the typing indicator), and I don't want to catch every single update related to last_msg (which would lead to useless additional bandwidth / realtime message cost)

I couldn't find anything about it in the documentation, something like below for example:
![image](https://github.com/supabase/supabase/assets/17619412/13667890-da4b-48e1-8d57-0261bf9d66ca)

or better yet, to listen to every other field but a specific one (again, trying to not catch all last_msg update)

**Comment by GaryAustin1** [comment]
No, there is no such feature.
